synfig-studio/plugins/lottie-exporter/common/Gradient.py

```diff
@@ -0,0 +1,211 @@
+"""
+Gradient.py
+Will store the gradient class
+"""
+
+import sys
+import copy
+import math
+import common
+import settings
+from common.Color import Color
+sys.path.append("..")
+
+
+class Gradient:
+    """
+    To store the gradients in Synfig and operations on them
+    Currently, it is assumed that only 2 colors will be provided in the gradient
+    """
+
+    def __init__(self, gradient):
+        """
+        Args:
+            gradient (lxml.etree._Element) : Gradient as extracted from the xml
+
+        Returns:
+            (None)
+        """
+        self.synfig_gradient = gradient
+        self.colors = []
+        self.extract_colors()
+
+    def extract_colors(self):
+        """
+        Helps in extracting all the colors present in the gradient
+        """
+        initial_list = []
+        for col in self.synfig_gradient:
+            red = float(col[0].text) ** (1/settings.GAMMA[0])
+            green = float(col[1].text) ** (1/settings.GAMMA[1])
+            blue = float(col[2].text) ** (1/settings.GAMMA[2])
+            alpha = float(col[3].text)
+            initial_list.append([float(col.attrib["pos"]), Color(red, green, blue, alpha)]) 
+        self.colors = initial_list
+
+    def reverse_gamma(self, color):
+        """
+        Given a color, it reverses the effect of gamma as done in extract_colors()
+
+        Args:
+            color (common.Color.Color) : color element
+
+        Returns:
+            (common.Color.Color) : color element with gamma effect reversed
+        """
+        ret = copy.deepcopy(color)
+        ret.red = ret.red ** settings.GAMMA[0]
+        ret.green = ret.green ** settings.GAMMA[1]
+        ret.blue = ret.blue ** settings.GAMMA[2]
+        return ret
+
+    def get_lottie_final_list(self):
```

Comment:
I don't think this function is used anywhere, but you can keep it for future use.